src/controllers/employee.controller.js

```diff
@@ -1,8 +1,16 @@
 const Employee = require("../models/employee.model")
-module.exports.addEmployee = (req,res) => {
-    if (req.body.FirstName.trim().length === 0
-        || req.body.LastName.trim().length === 0
-        || req.body.Status.trim().length === 0
+module.exports.addEmployee = async (req,res) => {
+    let contact = req.body.Contact.trim()
```

Comment:
In Javascript we use camel-casing:
SO you need to change variable names from:
Contact => contact

Reply:
done